### PR TITLE
Assignment 2 Submission

### PR DESCRIPTION
**Pull request by eternal-loading-screen**
## What changes are you trying to make? (e.g. Adding or removing code, refactoring existing code, adding reports)
Add assignment 2 answers

## What did you learn from the changes you have made?
to use the linear regression model

## Were there any challenges? If so, what issue(s) did you face? How did you overcome it?
handling the multiples  variables in the linear model

## How were these changes tested?
run locally

## Checklist
- [ ] I can confirm that my changes are working as intended

### CODE REVIEW

File: 02_activities/assignments/assignment_2.ipynb
Comment (Aditya-k-23) [inline 02_activities/assignments/assignment_2.ipynb:1]: We always prefer having the training set as a majority and the test set as a minority, so our model has the best chance for success. In your train test split, you have created the test set out of 75% of the data and the train with just 25%. Can you please fix this and resubmit?

**Review by Aditya-k-23 — CHANGES_REQUESTED** [review]

File: 02_activities/assignments/assignment_2.ipynb
Comment (Aditya-k-23) [reply 02_activities/assignments/assignment_2.ipynb:1]: Great job on the rest of the assignment tho! 🚀

**Comment by eternal-loading-screen** [comment]
Thanks @Aditya-k-23 for the feedback. I made the changes, let me know if that works.

**Review by Aditya-k-23 — APPROVED** [review]
